### DISCUSSION TITLE
graphql-sse/node_modules/graphql-sse/lib/client.mjs:67

### DISCUSSION THREAD

**Discussion by baebsndeep** [enisdenjo/graphql-sse#26] [Q&A] [answered]
Getting this error when using `express` to start server and using example client code.

![graphql-sse](https://user-images.githubusercontent.com/16997832/169100591-b606653b-af0a-46fb-9dee-9259d433fe52.png)

When I change the code in graphql-sse/node_modules/graphql-sse/lib/client.mjs from 
`const fetchFn = (options.fetchFn || fetch);` to `const fetchFn = (options.fetchFn);` as a workaround to I get a new error saying `fetchFn` is not a function.

**Comment by enisdenjo** [comment, marked as answer]
NodeJS does not natively provide a `fetch` function, you have to provide it yourself through the [`fetchFn` client option](https://github.com/enisdenjo/graphql-sse/blob/master/docs/interfaces/ClientOptions.md#fetchfn). 

Please consult the [" Client usage in Node" recipe](https://github.com/enisdenjo/graphql-sse#node-client).
